<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>trackbarsForm</class>
 <widget class="QWidget" name="trackbarsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>422</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>772</width>
    <height>422</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>772</width>
    <height>422</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Puzzle stacker</string>
  </property>
  <widget class="QPushButton" name="back_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Powrót</string>
   </property>
  </widget>
  <widget class="QPushButton" name="next_button">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>360</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Dalej</string>
   </property>
  </widget>
  <widget class="QLabel" name="contour_image">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>711</width>
     <height>261</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>711</width>
     <height>261</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>710</width>
     <height>260</height>
    </size>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;TextLabel&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSlider" name="max_slider">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>40</y>
     <width>631</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="singleStep">
    <number>5</number>
   </property>
   <property name="sliderPosition">
    <number>70</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="min_slider">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>631</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximum">
    <number>255</number>
   </property>
   <property name="singleStep">
    <number>5</number>
   </property>
   <property name="value">
    <number>40</number>
   </property>
   <property name="sliderPosition">
    <number>40</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="tickPosition">
    <enum>QSlider::NoTicks</enum>
   </property>
  </widget>
  <widget class="QLabel" name="min_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Wartość min: 40</string>
   </property>
  </widget>
  <widget class="QLabel" name="max_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Wartość maks: 70</string>
   </property>
  </widget>
  <widget class="QLabel" name="description">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>751</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Dostosuj wartość parametrów, aby uzyskać dobre rezultaty</string>
   </property>
  </widget>
  <widget class="QPushButton" name="default_button">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>360</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Przywróc domyślne</string>
   </property>
  </widget>
  <widget class="QPushButton" name="help_button">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>360</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Pomoc</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="prompt_image.qrc"/>
 </resources>
 <connections/>
</ui>
